src/studio/config.c

```diff
@@ -238,6 +238,7 @@ static void setDefault(Config* config)
             .volume         = MAX_VOLUME,
             .vsync          = DEFAULT_VSYNC,
             .fullscreen     = false,
+            .emacsMode      = false,
```

Comment:
Could you pls move all the emacs mode declarations under `#if defined(BUILD_EDITORS)` definition, which will allow not including this option for special builds without editors?